libraries/app/include/graphene/app/api.hpp

```diff
@@ -518,6 +520,31 @@ namespace graphene { namespace app {
          application& _app;
          graphene::app::database_api database_api;
    };
+
+   /**
+    * @brief The custom_operations_api class exposes access to standard custom objects parsed by the
+    * custom_operations_plugin.
+    */
+   class custom_operations_api
+   {
+      public:
+         custom_operations_api(application& app):_app(app), database_api( std::ref(*app.chain_database()),
+               &(app.get_options()) ){}
+
+         /**
+          * @breif Get all stored objects of an account in a particular catalog
```

Comment:
Typo: br~~e~~i*e*f

Reply:
thanks, fixed in https://github.com/bitshares/bitshares-core/pull/1926/commits/2a6cd6166fabb625a36942308a72ea25ba5433b3